src/main/java/io/seqera/tower/cli/responses/RunCanceled.java

```diff
@@ -0,0 +1,18 @@
+package io.seqera.tower.cli.responses;
+
+public class RunCanceled extends Response{
```

Comment:
The command `towr --json runs cancel ...` fails on binary CLI.

Reply:
Can't replicate this error locally using the binary build.